pgcli/main.py

```diff
@@ -156,6 +157,7 @@ def __init__(self, force_passwd_prompt=False, never_passwd_prompt=False,
         self.now = dt.datetime.today()
 
         self.completion_refresher = CompletionRefresher()
+        self.formatter = TabularOutputFormatter()
```

Comment:
I'm mildly curious. Is there a reason for this to be an instance variable? 

Why not instantiate the TabularOutputFormatter() inside the format_output() and use it?